### PR TITLE
POC report.

### PR DESCRIPTION
**Pull request by Inreet-Kaur**
Please review all the metrics and let me know if I need to change anything. If you want to add an explanation to something please do so.

### CODE REVIEW

File: docs/projMngmnt/POC_Team_Contrib.tex [hunk 1]
```diff
@@ -24,158 +24,171 @@
 
 \section{Demo Plans}
 
-\wss{What will you be demonstrating}
+% \wss{What will you be demonstrating}
+
+The main risk of the project is related to the reduction of documentation overhead. This means the biggest risk is really if this project can make a significant reduction to overhead time and truly reduce patient wait times. Again, through elicitation and our domain experts, we will create an effective set of requirements to reduce the risk of the project and build something that is needed.\\
```
Comment (gkaur034) [inline docs/projMngmnt/POC_Team_Contrib.tex:29]: add 'that' after means in the second line

File: docs/projMngmnt/POC_Team_Contrib.tex [hunk 2]
```diff
@@ -24,158 +24,171 @@
...
+
+Looking at implementation details there are a few risks that come up, if these risks can be addressed or have a clearer roadmap that will make us more confident in the project. 
+
+\begin{itemize}
+  \item \textbf{Speech Input} -- A hospital or a clinic can be a loud place, in the event audio input is taken we need to ensure that it is clean and clear. This would mean essentially blocking outside noise.
```
Comment (gkaur034) [inline docs/projMngmnt/POC_Team_Contrib.tex:34]: add 'so' after 'taken' in second line.

**Review by gkaur034 — APPROVED** [review]
Looks good!

**Review by PKALXI — APPROVED** [review]

**Review by moamen79 — COMMENTED** [review]
Looks good

**Pull request merged**